src/App.jsx

```diff
@@ -1,7 +1,32 @@
+import { useEffect, useState } from "react";
+import Flip from "./components/Flip";
+
 function App() {
- return (
-  <div>React exercises</div>
- )
+  const [data, setData] = useState([]);
+  const [change, setChange] = useState(true);
+  useEffect(() => {
+    fetch("https://random-data-api.com/api/users/random_user?size=10")
+      .then((res) => res.json())
+      .then((res) => setData(res));
+  }, [change]);
+   const handleClick = () => setChange((prev) => !prev);
```

Comment:
I prefer`   const handleClick = () => setChange(!change)` is better than targeting a previous value, do you really need a previous value?